apps/api/src/teams/event-registrations/event-registrations.service.ts

```diff
@@ -58,11 +85,17 @@ export class TeamEventRegistrationsService {
      */
 
     if (query.eventId) {
-      const event = await this.teamEventRepository.findOneOrFail({ teamEventId: query.eventId }, { populate: ['team', 'team.members'] });
+      const event = await this.teamEventRepository.findOneOrFail(
+        { teamEventId: query.eventId },
+        { populate: ['team', 'team.members'] },
+      );
       if (!event.canEdit(user))
         throw new ForbiddenException('Cannot view registrations');
       filter = { ...filter, event: { teamEventId: query.eventId } };
-    } else if (query.userId && (user.roles.includes(Role.ClubManager) || user.roles.includes(Role.Admin))) {
+    } else if (
+      query.userId
+      && (user.roles.includes(Role.ClubManager) || user.roles.includes(Role.Admin))
+    ) {
```

Comment:
Pourquoi l'avoir changé
```suggestion
    } else if (query.userId && (user.roles.includes(Role.ClubManager) || user.roles.includes(Role.Admin))) {
```